src/components/FormComponents/CustomFormComponents/Combobox.tsx

```diff
@@ -0,0 +1,86 @@
+import { BodyShort, Label } from '@navikt/ds-react';
+import React, { HTMLAttributes, PropsWithChildren } from 'react';
+import {
+    Combobox,
+    ComboboxInput,
+    ComboboxPopover,
+    ComboboxOption,
+    ComboboxInputProps,
+    ComboboxPopoverProps,
+    ComboboxProps,
+    ComboboxOptionProps,
+} from '@reach/combobox';
+import cn from 'clsx';
+
+import styles from './Combobox.module.css';
+
+export function ComboboxWrapper({
+    id,
+    label,
+    children,
+}: PropsWithChildren<{ id?: string; label: string }>): JSX.Element {
+    return (
+        <div className="navds-form-field navds-form-field--medium">
+            <Label id={id}>{label}</Label>
+            {children}
+        </div>
+    );
+}
+
+export function DsCombobox({
+    children,
+    className,
```

Comment:
Tenker vi treng ikkje legge til props før vi trenge det, slik som `className` i desse komponentane her. Men det skal vel kanskje litt meir styling til etterkvart, så reknar med det blir tatt i bruk.

Reply:
hmm ja godt poeng

Reply:
hmm eller problemet her er at `className`er definert i `HTMLAttributes<HTMLFooElement>`, som gjør at om jeg ikke sender det til `cn()` så vil de være i typedefinisjonen, men bare i all stillhet ikke bli applyet. Er redd for at det kan bli en sånn head-scratcher i fremtiden.